packages/angular/src/executors/module-federation-dev-server/lib/build-static-remotes.ts

```diff
@@ -1,7 +1,10 @@
 import { type Schema } from '../schema';
-import { logger, type ExecutorContext } from '@nx/devkit';
-import { fork } from 'child_process';
+import { type ExecutorContext, logger } from '@nx/devkit';
 import type { StaticRemotesConfig } from './parse-static-remotes-config';
+import { projectGraphCacheDirectory } from 'nx/src/utils/cache-directory';
+import { fork } from 'node:child_process';
+import { join } from 'node:path';
+import { createWriteStream } from 'node:fs';
```

Comment:
Will we run into issues with this style of import for users on older versions of Node?

Reply:
It's supported since Node 16, and we already use it in a few places.